melos.yaml

```
@@ -0,0 +1,45 @@
+name: amplify_flutter
+
+packages:
+  - packages/**
+
+scripts:
+  copy_dummy_config: >
+    melos exec --scope="*example*" --fail-fast -- \
+      cp ../../../.circleci/dummy_amplifyconfiguration.dart lib/amplifyconfiguration.dart
+
+  build:examples:ios: >
+    melos exec -c 1 --scope="*example*" --fail-fast -- \
+      flutter build ios --no-codesign
+
+  build:examples:android: >
+    melos exec -c 1 --scope="*example*" --fail-fast -- \
+      flutter build apk --debug
+
+  test:unit:flutter: >
+    ./.circleci/test_all_plugins.sh flutter-test && exit 0
+
+  test:unit:android: >
+    ./.circleci/test_all_plugins.sh android-test && exit 0
+
+  format: >
+    melos exec -c 1 -- \
+      flutter format .
+
+  analyze: >
+    melos exec -c 1 --no-private --ignore="*example*" -- \
+      pub global run tuneup check
+
+  lint:pub: >
+    melos exec -c 5 --fail-fast --no-private --ignore="*example*" -- \
+      pub publish --dry-run
+
+  postbootstrap: >
+    tuneup --version || pub global activate tuneup
```

```suggestion
    (tuneup --version || pub global activate tuneup) && melos run copy_dummy_config
```

Suggesting this so that it runs the copy on after bootstrap, otherwise there's a bunch of analyser errors in the examples as these files don't exist.

---

Thank you! This is great advice!